File: Day 6 : Let's Review/Solution.js
function processData(input) {
    inputSplitted = input.split("\n");
    for(let i = 1; i < inputSplitted.length; i++) {
         let even = "", odd = "";
        for(let j = 0; j < inputSplitted[i].length; j++) {
            (j % 2 == 0) ? even += inputSplitted[i][j] : odd += inputSplitted[i][j];
        }
        console.log(even, odd);
    }
}

main();

File: Day 6 : Let's Review/Solution.rb
T = gets.to_i

T.times do 
  str = gets.chomp
  even = Array.new
  odd = Array.new
  
  str.split("").to_a.each_with_index do |x,i|
      i % 2 == 0 ? even.push(x) : odd.push(x)
  end

  puts "#{even.join} #{odd.join}" 
end
